flytekit/clis/sdk_in_container/serialize.py

```diff
@@ -123,6 +137,19 @@ def get_registrable_entities(ctx: flyte_context.FlyteContext) -> typing.List:
 
     new_api_model_values = list(new_api_serializable_entities.values())
     entities_to_be_serialized = list(filter(_should_register_with_admin, new_api_model_values))
+    serializable_tasks: typing.List[task_models.TaskSpec] = [
+        entity for entity in entities_to_be_serialized if isinstance(entity, task_models.TaskSpec)
+    ]
+    # Detect if any of the tasks is duplicated. Duplicate tasks are defined as having the same metadata identifiers
+    # (see :py:class:`flytekit.common.core.identifier.Identifier`). Duplicate tasks are considered invalid at registration
```

Comment:
There's no need to do RST links in `#` comments - only text in docstrings `""" blah """` get rendered I'm pretty sure.